versioned_docs/version-1.0/cadence-migration-guide/usdc-migration.mdx

```diff
@@ -0,0 +1,336 @@
+---
+title: USDC Migration Guide
+sidebar_position: 8
+sidebar_label: USDC Migration Guide
+---
+
+# USDC in Cadence 1.0
+
+As part of the upcoming Flow Crescendo network upgrade, all ERC-20 compatible tokens on Flow
+can be used in Cadence smart contracts without special handling
+and all Cadence fungible tokens on Flow will be ERC-20 compatible.
+This will ensure that all Flow applications—whether they are built with
+Solidity, Cadence, or some mix of both—are fully interoperable with the wider web3 ecosystem.
+
+Effective September 3, 2024, at 3 a.m. ET ahead of the Crescendo network upgrade,
+[Circle will cease support for USDC on Cadence](https://www.flow.com/post/stablecoins-on-flow-evolving-for-interoperability),
+as EVM-based ERC-20 USDC will become naturally compatible across the entire Flow ecosystem.
+
+This means that the existing USDC (known as `FiatToken`) contract will not be upgraded
+to Cadence 1.0 as part of the Crescendo migration.
+[Balances can still be queried](https://github.com/onflow/flips/pull/283)
+and the broken vaults can still be moved around, but existing complex functionality like
+`withdraw()`, `deposit()`, and others will no longer work.
+
+To facilitate a smoother transition away from the old `FiatToken` contract,
+a new fungible token contract called `USDCFlow` is deployed on [testnet](https://testnet.flowdiver.io/contract/A.4516677f8083d680.USDCFlow?tab=deployments)
+and mainnet (COMING SOON).
+You can see the Github repo with the contract code and transactions to use it [here](https://github.com/onflow/bridged-usdc).
+that allows anyone with a `FiatToken` balance to deposit into the `USDCFLow` contract
+and mint the exact same amount of `USDCFlow`
+using the public `USDCFlow.wrapFiatToken()` function
+
+During the Crescendo migration, the `USDCFlow` contract will become part of the
+Flow VM bridge protocol and become the bridged version of USDC on Flow EVM.
+This USDC will be backed by real USDC from Ethereum mainnet, ensuring that 
+`USDCFlow` will seamlessly retain its 1 USD value and be able to be redeemed
+with Circle by bridging back to Ethereum mainnet.
+
+This document is primarily focused on what developers who rely on `FiatToken`
+can do to migrate to `USDCFlow` with as little disruption as possible.
+A brief guide is provided in the next section for regular USDC users
+about what they can do to migrate to the new token, but those users should primarily
+follow the [Flow Blog](https://www.flow.com/authors/flow) and communication
+from the apps they use about what their options are to migrate to the new USDC on Flow.
+
+# What can regular USDC holders do?
+
+If you are a user with USDC in your account, the recommended path is to directly 
+swap your USDC with `USDCFlow` using this transaction. This functionality
```

Comment:
What is 'this transaction'?

Reply:
It's not possible to do anything till it's deployed on mainnet right? Otherwise I can make a small video tutorial we can add.

Reply:
oops, meant to link to the transaction to use. I updated it.

Reply:
Yeah, they can't do anything on mainnet until it is deployed, but that is happening very soon